.github/workflows/userbenchmark-a100-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release TorchBench Userbenchmark on A100
+on:
+  pull_request:
+    paths:
+      - userbenchmark/release-test/*
+
+jobs:
+  run-userbenchmark:
+    runs-on: [a100-runner]
+    timeout-minutes: 1440 # 24 hours
+    environment: docker-s3-upload
+    env:
+      BASE_CONDA_ENV: "torchbench"
+      CONDA_ENV: "userbenchmark-a100"
+      PLATFORM_NAME: "gcp_a100"
+      SETUP_SCRIPT: "/workspace/setup_instance.sh"
+    steps:
+      - name: Checkout TorchBench
+        uses: actions/checkout@v3
+        with:
+          path: benchmark
+      - name: Tune Nvidia GPU
+        run: |
+          sudo nvidia-smi -pm 1
+          sudo nvidia-smi -ac 1215,1410
+          nvidia-smi
+      - name: Clone and setup conda env
+        run: |
+          CONDA_ENV=${BASE_CONDA_ENV} . "${SETUP_SCRIPT}"
+          conda create --name "${CONDA_ENV}" --clone "${BASE_CONDA_ENV}"
+      - name: Install TorchBench
+        run: |
+          set -x
+          . "${SETUP_SCRIPT}"
+          pushd benchmark
+          python install.py
+      - name: Run user benchmark
+        run: |
+          set -x
+          . "${SETUP_SCRIPT}"
+          # remove old results
+          release_version=$(cat ./.userbenchmark/release-test/version.txt)
```

Comment:
It seems `version.txt` is under `userbenchmark/release-test/version.txt`?

Reply:
Yes true, let me try